### DISCUSSION TITLE
Whether all features available for being self-hosted e.g. storage?

### DISCUSSION THREAD

**Discussion by icecoconut** [supabase/supabase#1998] [Questions] [answered]
Whether all features available for being self-hosted e.g. storage?

**Comment by inian** [comment, marked as answer]
You can find PR for storage [here](https://github.com/supabase/supabase/pull/1007). It is not merged in yet since it is not completely local and it needs an external S3 bucket, so I am trying to find if there is a way around that.

For dashboard, refer to https://github.com/supabase/supabase/discussions/1001#discussioncomment-558696

**Reply by mingfang** [reply]
I usually run Minio (https://github.com/minio/minio) locally as a S3 replacement.
